scripts/run_integrationtests.sh

```diff
@@ -22,6 +22,8 @@ docker build -t modynbase -f docker/Base/Dockerfile --build-arg MODYN_BUILDTYPE=
 docker compose up --build tests --abort-on-container-exit --exit-code-from tests
 exitcode=$?
 
+docker logs supervisor
```

Comment:
Leftover debugging?

Reply:
Yes, I plan to remove it as the last step before merging this PR. It helps me understand the github checks results.

Reply:
done